### PR TITLE
Update README as per current code base

### LINKED ISSUE #21: Detailed documentation in README
- Update the README with all detailed steps to use image builder.

### PR DESCRIPTION
**Pull request by balandi1**
- Merging if this PR closes Issue 21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,39 +2,14 @@
 Image builder is an application to build docker image for running assignments. The image is built using user specific configurations.
 
 ## Configurations
```
Comment (pradykaushik) [inline README.md]: make it singular.

File: README.md [hunk 2]
```diff
@@ -2,39 +2,14 @@
...
-    && make
-port:
-  number: 8082
-cmd:
-  params:
-    - ./code-runner/code-runner-server
-    - -port
-    - 8082
-```
+- The configurations required for assignment environment is provided in yaml format.
```
Comment (pradykaushik) [inline README.md]: singular.

File: README.md [hunk 3]
```diff
@@ -2,39 +2,14 @@
...
+- A docker image for [code-runner](https://github.com/assignment-exec/code-runner) application is used as base image for any assignment environment.
```
Comment (pradykaushik) [inline README.md]: as **the** base image

File: README.md (hunk 1 above)
Comment (pradykaushik) [reply README.md]: do this everywhere in the docs.

File: README.md [hunk 4]
```diff
@@ -48,6 +23,19 @@ dependencies:
...
 
+## Supported Languages
+Below is the list of supported languages and the corresponding versions.
+- gcc 7
+- g++ 7
+- python 3.7
+- java 8 & 11
+
+### Installation scripts
+- Every supported language and its version has an installation script stored in `scripts` directory.
```
Comment (pradykaushik) [inline README.md]: `[scripts](./scripts)`

File: README.md [hunk 5]
```diff
@@ -48,6 +23,19 @@ dependencies:
...
+- The scripts are named as `<language_version>.sh`. Example - For language - java and version - 8, script name should be `java_8.sh`.
+- The installation scripts are shell scripts that hold commands to install the corresponding language and its version.
```
Comment (pradykaushik) [inline README.md]: this statement is redundant to the first.

File: README.md [hunk 6]
```diff
@@ -48,6 +23,19 @@ dependencies:
...
+- To add support for a new language and version, add a new shell script that follows thw above given naming convention and holds the appropriate commands for installation.
```
Comment (pradykaushik) [inline README.md]: ~thw~ the

**Review by pradykaushik — APPROVED** [review]
Overall LGTM!
Minor nit picks to be resolved.

File: README.md (hunk 1 above)
Comment (balandi1) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (balandi1) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (balandi1) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (balandi1) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (balandi1) [reply README.md]: Removed

File: README.md (hunk 6 above)
Comment (balandi1) [reply README.md]: Done

**Pull request merged**